client/viewer/js/main.js

```diff
@@ -149,11 +150,24 @@ window.screensInit = function screensInit() {
 			// stop timers
 			carousel.destroy();
 			carousel = null;
+			DOM.carouselCountdown.style.transform = 'scaleX(0)';
+			DOM.carouselCountdown.style.transition = 'none';
+			DOM.carouselCountdown.style.offsetHeight;
 		}
 
 		if (Carousel.isCarousel(url)) {
 			carousel = new Carousel(url, host);
-			carousel.on('change', updateUrl);
+			carousel.on('change', function (url) {
+				updateUrl(url);
+				DOM.carouselCountdown.style.transition = 'none';
+				DOM.carouselCountdown.style.transform = 'scaleX(1)';
+
+				setTimeout(() => {
```

Comment:
To leave time for the css to flush. It'seems bad but there is no way to force this. I tried for ages for a neater solution.